app/models/publisher.rb

```diff
@@ -164,13 +174,18 @@ def email_verified?
     email.present?
   end
 
+  # Silly method for showing a color for people's avatar
+  def avatar_color
+    Digest::MD5.hexdigest(email || pending_email)[0...6]
```

Comment:
neat 😎

Reply:
<img src="https://media.giphy.com/media/SKGo6OYe24EBG/giphy.gif"/>

it's magic 🌈